ui_framework/doc_site/src/views/form/form_example.js

```diff
@@ -9,58 +10,76 @@ import {
   GuideText,
 } from '../../components';
 
-const textInputHtml = require('./text_input.html');
-const labelHtml = require('./label.html');
 const assistedInputHtml = require('./assisted_input.html');
 const searchInputHtml = require('./search_input.html');
 const staticInputHtml = require('./static_input.html');
-const textAreaHtml = require('./text_area.html');
-const textAreaNonResizableHtml = require('./text_area_non_resizable.html');
-const checkBoxHtml = require('./check_box.html');
-const selectHtml = require('./select.html');
+const Label = require('./label');
+const labelSource = require('!!raw!./label');
+const labelHtml = renderToHtml(Label);
+const TextInput = require('./text_input');
+const textInputSource = require('!!raw!./text_input');
+const textInputHtml = renderToHtml(TextInput, { id: '1' });
+const TextArea = require('./text_area');
+const textAreaSource = require('!!raw!./text_area');
+const textAreaHtml = renderToHtml(TextArea);
+const TextAreaNonResizable = require('./text_area_non_resizable');
+const textAreaNonResizableSource = require('!!raw!./text_area_non_resizable');
+const textAreaNonResizableHtml = renderToHtml(TextAreaNonResizable);
+const Select = require('./select');
+const selectSource = require('!!raw!./select');
+const selectHtml = renderToHtml(Select);
+const CheckBox = require('./check_box');
+const checkBoxSource = require('!!raw!./check_box');
+const checkBoxHtml = renderToHtml(CheckBox);
```

Comment:
Could you do me one last favor and use line breaks to group these variables?

```javascript
const assistedInputHtml = require('./assisted_input.html');

const searchInputHtml = require('./search_input.html');

const staticInputHtml = require('./static_input.html');

const Label = require('./label');
const labelSource = require('!!raw!./label');
const labelHtml = renderToHtml(Label);

const TextInput = require('./text_input');
const textInputSource = require('!!raw!./text_input');
const textInputHtml = renderToHtml(TextInput, { id: '1' });

const TextArea = require('./text_area');
const textAreaSource = require('!!raw!./text_area');
const textAreaHtml = renderToHtml(TextArea);

const TextAreaNonResizable = require('./text_area_non_resizable');
const textAreaNonResizableSource = require('!!raw!./text_area_non_resizable');
const textAreaNonResizableHtml = renderToHtml(TextAreaNonResizable);

const Select = require('./select');
const selectSource = require('!!raw!./select');
const selectHtml = renderToHtml(Select);

const CheckBox = require('./check_box');
const checkBoxSource = require('!!raw!./check_box');
const checkBoxHtml = renderToHtml(CheckBox);
```

Reply:
Ok, it has sense.